src/cloudEnvs/openLogin.ts

```diff
@@ -3,8 +3,12 @@ import { URL } from "url";
 import { StateType, TABNINE_URL_QUERY_PARAM } from "../globals/consts";
 import createHubWebView from "../hub/createHubWebView";
 import hubUri from "../hub/hubUri";
+import {isSandboxed} from "../sandbox";
 
 export default async function openLogin(): Promise<void> {
+  if (isSandboxed()) {
```

Comment:
Is this enough? 
I think we want to remove the option altogether

Reply:
If you mean in respect to a user interaction then yes. but there are internal code paths which lead to hear and this will block that